osbuild/inputs.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, name, info, origin: str, options: Dict):
         self.name = name
         self.info = info
         self.origin = origin
-        self.refs = {}
+        self.refs: Dict[str, Optional[Dict[str, Any]]] = {}
```

Comment:
The code below (`self.refs[ref] = options or {}`) would mean this is `Dict[str, Dict[str, Any]]`, no?